syntax = "proto3";
package nlp;

service Nlp {
    rpc Ner (NerRequest) returns (NerReply);
}

message NerRequest {
    string text = 1;
    string field = 2;
}

message NerReply {
    string ner_ret = 1;
}
